training/creating_dataset.py

```diff
@@ -92,13 +288,45 @@ def load_and_preprocess_data(
     df = bestiary[raw_names]
     df.columns = target_names
 
+    for resistance in characteristics_groups.resistances:
+        characteristic_name = resistance.replace("_resistance", "")
+        get_value = lambda x: get_characteristic_from_list(x, characteristic_name)
+        df[resistance] = bestiary[RESISTANCE_PATH].apply(get_value)
+
+    for weakness in characteristics_groups.weaknesses:
+        characteristic_name = weakness.replace("_weakness", "")
+        get_value = lambda x: get_characteristic_from_list(x, characteristic_name)
+        df[weakness] = bestiary[WEAKNESSES_PATH].apply(get_value)
+
+    for speed in characteristics_groups.speeds:
+        get_value = lambda x: get_characteristic_from_list(x, speed)
+        df[speed] = bestiary[OTHER_SPEED_PATH].apply(get_value)
```

Comment:
Te trzy fory (for resistance, for weakness i for speed) są do siebie bardzo podobne - czy nie dałoby się stworzyć funkcji, która byłaby wywoływana trzy razy zamiast tak duplikować kod? Chyba że okazałoby się to niewygodne, to wtedy może zostać jak jest

Reply:
To wsm mogę dać do osobnej funkcji. 
// Chyba nawet sama o tym myślałam tydzień temu, ale oczywiści zapomniałam,😅  więc popieram